datafusion/core/tests/sqllogictests/test_files/window.slt

```diff
@@ -2642,8 +2642,9 @@ ProjectionExec: expr=[fv1@0 as fv1, fv2@1 as fv2, lv1@2 as lv1, lv2@3 as lv2, nv
 ------------CsvExec: file_groups={1 group: [[WORKSPACE_ROOT/datafusion/core/tests/data/window_1.csv]]}, projection=[ts, inc_col], output_ordering=[ts@0 ASC NULLS LAST], has_header=true
 
-query IIIIIIIIIIIIIIIIIIIIIIII
+query IIIIIIIIIIIIIIIIIIIIIIIII
 SELECT
+  ts,
```

Comment:
I added `ts` to this query to show that the the first two rows have a tie in the same value `264` and thus it is ok that the output order changes.  We should probably fix the test (or could add `rowsort`) but that might obscure real errors in the future 🤔